src/index.html

```diff
@@ -5,8 +5,319 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="top-bar">
+      <a href="#" class="top-bar__logo">
+        <img src="./images/logo.png" alt="logo"
+        class="img_logo">
+      </a>
+
+      <nav class="top-bar__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link nav__link--is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link "
+               data-qa="nav-hover">laptops & computers</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">tablets</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">photo</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog catalog--position">
+        <div
+           class="card"
+           data-qa="card">
+          <img
+            class="card__img"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina">
+
+          <div class="card__title title" >
+            <h2 class="title__text" >APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
Do not hardcode text new line
```suggestion
            <h2 class="title__text" >APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```